package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "realm",
   "description": "Realm is a mobile database: an alternative to SQLite and key-value stores",
-  "version": "10.0.0-beta.10",
+  "version": "10.0.0-beta.11",
```

Comment:
`10.0.0-beta.9` is the latest beta release on https://www.npmjs.com/package/realm & changelog marks updated as `10.0.0-beta.10`

Reply:
that's a mistake see the branch name :)